app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -417,7 +416,6 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
       ) {
         override fun onDrawerOpened(drawerView: View) {
           super.onDrawerOpened(drawerView)
-          fragment.activity!!.invalidateOptionsMenu()
```

Comment:
Why is it okay to remove this? You should source this back to the original PR where it was introduced to try and determine why it's here since removing it may have unintended consequences not caught by automated or manual testing.

Reply:
Hi @BenHenning ,
I found the fragment to be removed maybe due to a fragment transaction complete.
Due to which the crash happened. For the Drawer opening case also I did because of fragment could be null.
I will come back with more analysis